### DISCUSSION TITLE
Serverless Function Crashed with INTERNAL_SERVER_ERROR (Code: FUNCTION_INVOCATION_FAILED)

### DISCUSSION THREAD

**Discussion by h7ml** [vercel/community#2560] [Help]
### Summary

I have encountered an issue with my Next.js project deployed on Vercel. The project's GitHub repository can be found [here](https://github.com/h7ml/nestjs-nakoruru/tree/master), and the deployed URL is https://nestjs.h7ml.cn/. Until May 18th, the deployment was successful, but recently I have been experiencing an error. The error message displayed on the interface is as follows:

vbnet
```
This Serverless Function has crashed.

Your connection is working correctly.

Vercel is working correctly.

500: INTERNAL_SERVER_ERROR
Code: FUNCTION_INVOCATION_FAILED
ID: lhr1::9gqxt-1684847159862-840f9f3bd634

```
If you are a visitor, contact the website owner or try again later.
If you are the owner, learn how to fix the error and check the logs.
I'm reaching out to seek assistance in resolving this issue. It seems to be related to a serverless function crashing with an INTERNAL_SERVER_ERROR and FUNCTION_INVOCATION_FAILED code. The error ID provided is lhr1::9gqxt-1684847159862-840f9f3bd634.

Could you please help me understand the cause of this error and guide me on how to resolve it? I appreciate any insights or suggestions you can provide.

Thank you!

**Comment by amyegan** [comment]
Hi @h7ml! The [FUNCTION_INVOCATION_FAILED](https://vercel.com/docs/errors/application-error-codes#function_invocation_failed) error means there was an unhandled exception in the function. You may be able to find more information in the [function logs](https://vercel.com/docs/concepts/observability/runtime-logs). You can also log additional information and handle exceptions using a [try/catch statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch)
